conans/test/command/search_test.py

```diff
@@ -193,6 +195,8 @@ def package_search_properties_filter_test(self):
             arch: x86
             compiler: gcc
             compiler.version: 4.3
+        [recipe hash]
```

Comment:
I would say that this is a bit verbose and uninformative. What can I do with such hash?. What does it means?. I'd say that a single line something like: ``outdated from recipe: True/False`` could be better.